website/docs/d/nginx_deployment.html.markdown

```diff
@@ -63,7 +63,7 @@ In addition to the Arguments listed above - the following Attributes are exporte
 
 * `nginx_version` - NGINX version of the Nginx Deployment.
 
-* `sku` - Name of the SKU for this Nginx Deployment.
+* `sku` - (Required) Specifies the Nginx Deployment SKU. Possible values include `standard_Monthly`. Changing this forces a new resource to be created.
```

Comment:
this is a data source. this should be worded as a read field and not a settable field like the resource.

Reply:
Oops I missed this, thanks @puneetsarna

Reply:
no worries @manicminer and thank you for the quick fix 👍